test/test_point_inclusion.cpp

```diff
@@ -241,7 +241,6 @@ TEST(BoxPointContainment, Basic)
   EXPECT_FALSE(box.containsPoint(Eigen::Vector3d(0.00, 1.01, 1.01)));
 
   // near three-axis maximum
-  const double sq3 = sqrt(3) / 3;
```

Comment:
Weird, this seems to belong to the original test (see https://github.com/ros-planning/geometric_shapes/blob/melodic-devel/test/test_point_inclusion.cpp#L95). Probably a hotfix to get this test to pass. Could you just revert this change and fix the original test?

Reply:
Yes, these warnings also belong to the original test, which also showed up in the [MoveIt CI](https://travis-ci.org/ros-planning/moveit/jobs/610040749#L1264) result. I filed up a [PR](https://github.com/ros-planning/geometric_shapes/pull/117) to the original test, and temporarily disable these two warnings here until the original test is fixed.

Reply:
Oh, I see.. great catch!